docs/source/procedures/datadog/cloud-integration.rst

```diff
@@ -0,0 +1,60 @@
+=============================================
+ScyllaDB Cloud Monitoring Datadog Integration
+=============================================
+
+For security reasons, the ScyllaDB cloud does not have direct access to the Prometheus server.
```

Comment:
"For security reasons, the ScyllaDB cloud does not have direct access to the Prometheus server." This can be removed; it does not add useful information. We might move away from Prometheus in the future.